### PR TITLE
Added release cycle and versioning documentation

### PR DESCRIPTION
**Pull request by Gaptcha**
##### SUMMARY
This is the formatted release cycle and versioning documentation guideline based on the work in progress [document](https://docs.google.com/document/d/1SHL8XeCj9iUdfXohj1M92AQoASh6il9v5aFtSotoBrY/edit?usp=sharing).

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs/RELEASE.md

### CODE REVIEW

**Comment by D3DeFi** [comment]
@sky-joker @dj-wasabi @rockaut this ties to our discussions and ongoing WIP gdoc for this topic.

Related to the #171 and #26

Fixes #26

File: docs/RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Release cycle and versioning
+
+## Versioning
+Versioning is using SemVer (X.Y.Z):
+- The X is a **major version** and is incremented when:
```
Comment (D3DeFi) [inline docs/RELEASE.md:5]: From our comments: "This should only be incremented on breaking changes that are backwards incompatible" - I believe all of the items match that

File: docs/RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - Support for older Zabbix versions is removed.
+    - Support for older Ansible versions is removed.
+    - Support for older Python versions is removed.
+    - Modules, roles or plugins are removed.
+    - Module or role functionality is removed.
+    - Other breaking changes or backward-incompatible changes are introduced.
+
+- The Y is a **minor version** and is incremented when:
```
Comment (D3DeFi) [inline docs/RELEASE.md:13]: From our comments: "This is the main number we are focusing on and only do Z releases if we need to fix something quickly (e.g. right after we release Y and find bug"

File: docs/RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - Bugs are fixed in a backward-compatible way.
+    - Documentation fixes and smaller changes are introduced.
+## Releases
+Release dates are not fixed. Instead, they will be discussed at the beginning of each month following this guideline:
+- The version increment will depend on the content that will be included in the release, as discussed in the *Versioning* section.
+- New collection releases may be a result of this discussion if necessary.
+- There may be several releases during the month if needed.
+## Collection support
+The latest release of the community.zabbix is always supported.
+Older releases, which are included in the still supported Ansible versions, may obtain occasional backports or bug fixes when necessary. [1]
```
Comment (D3DeFi) [inline docs/RELEASE.md:31]: This, unfortunately, we need to do, since we are going to be a part of ansible releases.

**Comment by sky-joker** [comment]
> Edit: also this should be linked from README.md I guess, any tips on where it is best placed?

Sorry, I’m not sure, how about adding the Release policy section to README?
And add the link in the Release policy section?

**Comment by sky-joker** [comment]
Here's like?

```
(snip)
## Supported Zabbix versions
(snip)
## About Release Policy

If you want to check about the Release Policy of Zabbix collections.  
[link]

## License
(snip)
```

**Review by dj-wasabi — APPROVED** [review]

**Review by sky-joker — APPROVED** [review]

**Pull request merged**